<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multichat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
        background-color: #17212B;
    }
    QMenuBar {
        background-color: #1A252F;
        color: #E6ECEF;
        font-family: Roboto, Arial, sans-serif;
        font-size: 12pt;
        padding: 5px;
    }
    QMenuBar::item {
        background-color: #1A252F;
        color: #E6ECEF;
        padding: 5px 10px;
    }
    QMenuBar::item:selected {
        background-color: #5288C1;
    }
    QMenu {
        background-color: #1A252F;
        color: #E6ECEF;
        font-family: Roboto, Arial, sans-serif;
        font-size: 11pt;
        border: 1px solid #3F4B5A;
    }
    QMenu::item {
        padding: 5px 20px;
    }
    QMenu::item:selected {
        background-color: #5288C1;
        color: #FFFFFF;
    }
    QLabel {
        font-family: Roboto, Arial, sans-serif;
        font-size: 12pt;
        color: #E6ECEF;
    }
    QTextEdit {
        font-family: Roboto, Arial, sans-serif;
        font-size: 12pt;
        color: #E6ECEF;
        background-color: #2B3743;
        border: 1px solid #3F4B5A;
        border-radius: 5px;
        padding: 5px;
    }
    QLineEdit {
        font-family: Roboto, Arial, sans-serif;
        font-size: 12pt;
        background-color: #2B3743;
        color: #E6ECEF;
        border: 1px solid #3F4B5A;
        border-radius: 5px;
        padding: 5px;
    }
    QLineEdit:focus {
        border: 1px solid #5288C1;
    }
    QLineEdit::placeholder {
        color: #6A7886;
    }
    QPushButton {
        font-family: Roboto, Arial, sans-serif;
        font-size: 10pt;
        background-color: #5288C1;
        color: #FFFFFF;
        border: none;
        padding: 8px;
        border-radius: 5px;
    }
    QPushButton:hover {
        background-color: #6A9ED1;
    }
    QPushButton:pressed {
        background-color: #3F6A9B;
    }
    QPushButton#sendButton {
        min-width: 50px;
    }
    QListWidget {
        font-family: Roboto, Arial, sans-serif;
        font-size: 11pt;
        background-color: #2B3743;
        color: #E6ECEF;
        border: 1px solid #3F4B5A;
        border-radius: 5px;
        padding: 5px;
    }
    QListWidget::item {
        padding: 5px;
    }
    QListWidget::item:selected {
        background-color: #5288C1;
        color: #FFFFFF;
    }
    QStatusBar {
        font-family: Roboto, Arial, sans-serif;
        font-size: 10pt;
        color: #E6ECEF;
        background-color: #1A252F;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="mainLayout" stretch="0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="sidebarLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="peersLabel">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #E6ECEF; padding: 5px;</string>
          </property>
          <property name="text">
           <string>Группы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="peersList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="modeButtonsLayout">
          <item>
           <widget class="QPushButton" name="broadcastButton">
            <property name="text">
             <string>Broadcast-режим</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="multicastButton">
            <property name="text">
             <string>Multicast-режим</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sharedButton">
            <property name="text">
             <string>Shared-режим</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::EditDelete"/>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="infoButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="chatLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item>
         <widget class="QTextEdit" name="chatDisplay">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto','Arial','sans-serif'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="inputLayout">
          <item>
           <widget class="QLineEdit" name="messageInput">
            <property name="placeholderText">
             <string>Введите сообщение или команду...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::MailSend"/>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1246</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1A252F; color: #E6ECEF;</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionLeaveGroup"/>
    <addaction name="actionJoinGroup"/>
    <addaction name="actionPeersBroadcast"/>
    <addaction name="actionPeersMulticast"/>
    <addaction name="actionPeersIgnore"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCommands"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit (/exit)</string>
   </property>
  </action>
  <action name="actionLeaveGroup">
   <property name="text">
    <string>Leave Group (/leave_group)</string>
   </property>
  </action>
  <action name="actionJoinGroup">
   <property name="text">
    <string>Join Group (/join_group)</string>
   </property>
  </action>
  <action name="actionPeersBroadcast">
   <property name="text">
    <string>Show Broadcast Peers (/peers_broadcast)</string>
   </property>
  </action>
  <action name="actionPeersMulticast">
   <property name="text">
    <string>Show Multicast Peers (/peers_multicast)</string>
   </property>
  </action>
  <action name="actionPeersIgnore">
   <property name="text">
    <string>Show Ignored Peers (/peers_ignore)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
